cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -283,6 +283,16 @@ pKeyCmd =
             Opt.progDesc $ "Convert an Incentivized Testnet (ITN) non-extended "
                         ++ "signing or verification key to a corresponding "
                         ++ "Shelley stake key"
+      , Opt.command "convert-itn-extended-key" $
+          Opt.info pKeyConvertITNExtendedKey $
+            Opt.progDesc $ "Convert an Incentivized Testnet (ITN) extended "
+                        ++ "signing key to a corresponding "
+                        ++ "Shelley stake signing key"
+      , Opt.command "convert-itn-bip32-key" $
+          Opt.info pKeyConvertITNBip32Key $
+            Opt.progDesc $ "Convert an Incentivized Testnet (ITN) BIP32 "
+                        ++ "signing key to a corresponding "
+                        ++ "Shelley stake signing key"
```

Comment:
I wonder whether we should include the [terminology used on the ITN](https://input-output-hk.github.io/jormungandr/jcli/key.html) somewhere in the `progDesc`: `Ed25519Extended` and `Ed25519Bip32`.

This might be useful to a user familiar with the `jcli` key `--type` option.

Reply:
Yes. Useful to be able to clearly relate them.